web/src/types/global.js

```diff
@@ -3,4 +3,6 @@
  * This software is licensed under the Apache License version 2.
  */
 
-declare type Recordable<T = any> = Record<string, T>
+/**
+ * @typedef {Object.<string, any>} Recordable
+ */
```

Comment:
The files under web/src/types/*.js  are not necessary.

Reply:
Got it, so do you want me to delete both files? anxios.d.ts and global.d.ts?